code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -910,8 +910,8 @@
 		/obj/item/stack/sheet/glass = 1)
 	def_components = list(/obj/item/stack/ore/bluespace_crystal = /obj/item/stack/ore/bluespace_crystal/artificial)
 
-/obj/item/circuitboard/machine/dnascanner
-	name = "DNA Scanner (Machine Board)"
+/obj/item/circuitboard/machine/clonescanner
+	name = "Cloning Scanner (Machine Board)"
```

Comment:
I'm curious, what's the reason behind renaming this? Did you check to see if it wouldn't tamper with Genetics?

Reply:
I assume the clone scanner is just a improved version of the dna scanner, still probably better to keep both version or to swap the code from the clone scanner inside the dna scanner.
Having 2 machines still is the easiest way to go IMO.